sdmetrics/reports/base_report.py

```diff
@@ -80,10 +80,14 @@ def _validate(self, real_data, synthetic_data, metadata):
                 The metadata of the table.
         """
         self._validate_data_format(real_data, synthetic_data)
-        self._validate_metadata_matches_data(real_data, synthetic_data, metadata)
+        try:
+            self._validate_metadata_matches_data(real_data, synthetic_data, metadata)
 
-    def _handle_results(self, verbose):
-        raise NotImplementedError
+        except ValueError as e:
+            if self.__class__.__name__ == 'DiagnosticReport':
+                return
+
+            raise e
```

Comment:
Instead of a try/except can we just skip this check if it's a `DiagnosticReport`?

Reply:
Yes, done in [edfffc0](https://github.com/sdv-dev/SDMetrics/pull/524/commits/edfffc0717f903f04009b1833ebcb97479c3a238)